ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,23 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
-    libindy-crypto=0.4.5 \
     libindy=1.13.0~1420 \
+    libssl-dev \
 # rocksdb python wrapper
     libbz2-dev \
     zlib1g-dev \
     liblz4-dev \
     libsnappy-dev \
+    ursa=0.3.2-1  \
```

Comment:
BTW when I tried to install it on Ubuntu 16.04 by running
```
apt-get update 
apt-get install ursa=0.3.2-1
```
I got
```
 ursa : Depends: libssl1.1 (>= 1.1.1) but it is not installable
        Depends: libc6 (>= 2.27) but 2.23-0ubuntu11 is to be installed
        Depends: libgcc1 (>= 1:8.3.0) but 1:6.0.1-0ubuntu1 is to be installed
```

Reply:
I think there are several problems with current ursa package:
- **[critical]** it fails to install on Ubuntu 16.04 because of dependency on libssl1.1 (which is missing on 16.04), so in its current state it is basically unusable on current Indy Node main target platform
- for some reason deb package has dependencies both on on libssl1.0.0 and libssl1.1 - why it is so?
- not a problem now, but it also fails to install on Ubuntu 20.04 because of dependency on libssl1.0.0 (which is missing on 20.04)

Also Ursa deb package provided for xenial in repo.sovrin.org is same (their sha256 match) as one for bionic - with all above mentioned it looks like it was just built on Ubuntu 18.04 and then just copied to both locations.

To fix an issue I suggest to create a separate Ursa build for Ubuntu 16.04.

Reply:
Not sure whether it would help, but after manually extracting libursa.so from deb package I found (using ldd) that it seems it really depends only on libssl1.1, which means that in order to be usable in Ubuntu 20.04 binary can be just repacked into deb with correct dependencies. However, in order to make Ursa available on Ubuntu 16.04 you still need to rebuild libursa.so, because Ubuntu 16.04 doesn't have libssl1.1

Reply:
I believe this is now fixed and can work on ubuntu 16.04.   I will push the changes so that this can happen